docs/admin/misc_configuration_tips.md

```diff
@@ -19,6 +19,22 @@ If running Stellio in Kubernetes, it can be configured in the deployments:
   value: 512KB
 ```
 
+## Increase the default and maximum limit for pagination
+
+Stellio has a default pagination limit of 30 and a maximum of 100. 
+
+These values can be changed by configuring the `application.pagination.limit-default` and `application.pagination.limit-max` properties in `shared.properties` file.
+
+If running Stellio from `docker-compose`, it can be configured in the environment section of the root project : 
+
+```
+      - APPLICATION_PAGINATION_LIMIT_DEFAULT=30
+      - APPLICATION_PAGINATION_LIMIT_MAX=100
```

Comment:
they do not have this name in the `docker-compose.yml` file provided in Stellio GH...

Reply:
I was referencing the variables in the `.env` file

Reply:
here you are not referencing them, don't see the point then.

Reply:
in the sentence before it says : "it can be configured in the environment section of the target service" so referencing the variables in the `docker-compose.yml` would be confusing no ?

Reply:
`APPLICATION_PAGINATION_LIMIT-DEFAULT` (in Stellio GH) vs `APPLICATION_PAGINATION_LIMIT_DEFAULT` (here)